### ISSUE TITLE
TensorFlow.yaml common capabilities and common patterns supported cases.

### ISSUE THREAD

**Issue by cfRod** [intel/lpot#3]
Hi,

I have a couple of questions to help me understand the quantization flow and the  supported configurations in the TensorFlow.yaml file provided.

- Under common capabilities: for the activation dtype here: why isn't int8 supported? https://github.com/intel/lpot/blob/6eea6bdc98a888b0d6dfd88598786c0a2f3e2a47/lpot/adaptor/tensorflow.yaml#L26
- Under common patterns: in the int8 category: why isn't there support for Relu6?  
https://github.com/intel/lpot/blob/6eea6bdc98a888b0d6dfd88598786c0a2f3e2a47/lpot/adaptor/tensorflow.yaml#L66
- If I wanted to generate an int8 model using PTQ that could ultimately be of src,wei,dst=u8s8u8  for all convolutional layers. Can this be enforced by the user - in terms of search space?
```
dnnl_verbose,exec,cpu,reorder,jit:uni,undef,src_f32::blocked:acdb:f0 dst_s8::blocked:acdb:f0,oscale:0:0.840726;,,1x224x224x3,0.032959
dnnl_verbose,exec,cpu,convolution,jit_int8:avx512_core,forward_training,src_s8::blocked:acdb:f0 wei_s8:p:blocked:ABcd4b16a4b:f3 bia_f32::blocked:a:f0 dst_u8::blocked:acdb:f0,oscale:2;post_ops:'eltwise_relu;';,alg:convolution_direct,mb1_ic3oc64_ih224oh112kh7sh2dh0ph3_iw224ow112kw7sw2dw0pw3,0.184082
dnnl_verbose,exec,cpu,pooling,jit_int:avx512_core,forward_inference,src_u8::blocked:acdb:f0 dst_u8::blocked:acdb:f0 ws_undef::undef::f0,,alg:pooling_max,mb1ic64_ih112oh56kh3dh0sh2ph0_iw112ow56kw3dw0sw2pw0,0.0258789
dnnl_verbose,exec,cpu,convolution,jit_int8_1x1:avx512_core,forward_training,src_u8::blocked:acdb:f0 wei_s8::blocked:ABcd4b16a4b:f0 bia_s32::blocked:a:f0 dst_s8::blocked:acdb:f0,oscale:2;,alg:convolution_direct,mb1_ic64oc256_ih56oh56kh1sh1dh0ph0_iw56ow56kw1sw1dw0pw0,0.0419922
dnnl_verbose,exec,cpu,convolution,jit_int8_1x1:avx512_core,forward_training,src_u8::blocked:acdb:f0 wei_s8::blocked:ABcd4b16a4b:f0 bia_s32::blocked:a:f0 dst_u8::blocked:acdb:f0,oscale:2;post_ops:'eltwise_relu;';,alg:convolution_direct,mb1_ic64oc64_ih56oh56kh1sh1dh0ph0_iw56ow56kw1sw1dw0pw0,0.0249023
```

**Comment by ftian1** [comment]
Thanks, cfRod.

We will fix the activation int8 issue and Conv + [BiasAdd] + Relu6 missing bug at next release.

currently intel optimized TensorFlow CPU path is using a oneDNN old version, which only support symmetric quantization on conv op. so there is no way to enforce u8s8u8 for all conv.

**Comment by cfRod** [comment]
Hi ftian1,

Thanks for your reply. With the move of TensorFlow2.5.0 to a newer oneDNN 2.2. Would  there be scope to use/add functionality to lpot to generate a model that enforces u8s8u8 for the conv ops?

**Comment by hshen14** [comment]
Thanks for reporting the issues. Fixed in commit a7e2f0ff13cd8ed4cf06e56b398e28ed5187dbbe. Closing the issue.
